prusti-contracts/prusti-contracts/src/core_spec/clone.rs

```diff
@@ -0,0 +1,12 @@
+use crate::*;
+
+#[extern_spec]
+trait Clone {
+    #[ghost_constraint(Self: SnapshotEqualClone, [
+		ensures(result === self),
+	])]
```

Comment:
Use spaces throughout.

Reply:
Ah right, rustfmt doesn't touch whitespace within attributes because it treats them opaquely. I'll find-replace all tabs with 4 spaces, and try to keep in mind to check occasionally since they seem to like to slip in.